src/test/java/teammates/test/cases/action/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -1126,7 +1128,8 @@ protected void testAccessControl() throws Exception {
 
     private void testGracePeriodAccessControlForStudents() {
         FeedbackSessionAttributes fs = typicalBundle.feedbackSessions.get("gracePeriodSession");
-        fs.setEndTime(TimeHelper.convertLocalDateToUtc(TimeHelper.getDateOffsetToCurrentTime(0), fs.getTimeZone()));
+        fs.setEndTime(TimeHelper.convertLocalDateTimeToInstant(
+                TimeHelperExtension.now(), TimeHelper.convertToZoneId(fs.getTimeZone())));
```

Comment:
This logic seems long winded, probably a result of the fact that these times used to be in local time instead of UTC. Just `fs.setEndTime(Instant.now())` would be sufficient here. Personally, I find `TimeHelperExtension.now` very awkward, i.e. it shouldn't be there. Wherever it's used, I feel like it's because of the long-winded logic such as this.

Reply:
You're right; just tried and realised it isn't necessary for this test. However it's still needed in `StudentHomePageUiTest` due to a problem with the test itself. In that test, the end time of `Graced Feedback Session` is set to `x` hours before the current time where `x` is the zone offset for the session, such that when displayed in the user's time zone, it is printed as the current time in UTC. This is necessary for the test to pass due to broken time zone detection in `HtmlHelper`, where it searches for the string `UTC+xxxx` to detect the time zone. However, that particular page has no such string, so the zone is detected as UTC, and the search and replace for `datetime.now` only matches the current time in UTC. Since this is a problem with the test, I will explicitly set it as such, and it can be fixed in a future PR. Will remove `TimeHelperExtension#now`.